src/main/java/seedu/address/storage/JsonAdaptedUuid.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.storage;
+
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+/**
+ * Jackson-friendly version of {@link java.util.UUID}.
+ */
+class JsonAdaptedUuid {
+
+    public static final String MESSAGE_NULL_UUID = "UUID is null";
+
+    private final String uuid;
```

Comment:
Naming here might be a bit confusing since uuid is a object type itself.
```suggestion
    private final String uuidString;
```